d3-geo/index.d.ts

```diff
@@ -936,6 +936,16 @@ export interface GeoPath<This, DatumObject extends GeoPermissibleObjects> {
     centroid(object: DatumObject): [number, number];
```

Comment:
Can you please insert `object` after `@param` to read `@param object An Object...`. I must have overlooked the omission, when adding the JSDoc comments originally.

The same omission is also there for the `area` and `bounds` method. Thx.
Since we already have a PR open... 😄